### PR TITLE
Skip link and regions

### PR DESCRIPTION
**Pull request by KirillAbramo**
For accessibility purpose two features were implemented:
1. Skip to main content link - which allows user to skip site navigation menu with Tab click to go directly to main content on site pages.
Link was implemented in app component. It appears on the site with the first Tab hit, if user click on it moves focus from navigation items and leads to the anchors main-content of site pages. It works by setting focus on skip-link with pages onloading and then if clicked focusing on last-focus element of app component.
2. Page regions were implemented to provide better screen-reader users experience. 
Main tag was implemented on site pages to hold main content, Role attribute was added to Header, Footer, Main tags, Aria-labels were set up for header and footer Navigation menus.

### CODE REVIEW

**Comment by luchward** [comment]
1. You have to add description.
2. Execute "npm run stylelint" and fix style bugs or code smell before pushing branch to remote repository.

**Review by luchward — CHANGES_REQUESTED** [review]
Add description

**Comment by KirillAbramo** [comment]
> Add description

done

**Comment by KirillAbramo** [comment]
For accessibility purpose two features were implemented:

Skip to main content link - which allows user to skip site navigation menu with Tab click to go directly to main content on site pages.
Link was implemented in app component. It appears on the site with the first Tab hit, if user click on it moves focus from navigation items and leads to the anchors main-content of site pages. It works by setting focus on skip-link with pages onloading and then if clicked focusing on last-focus element of app component.
Page regions were implemented to provide better screen-reader users experience.
Main tag was implemented on site pages to hold main content, Role attribute was added to Header, Footer, Main tags, Aria-labels were set up for header and footer Navigation menus.

File: src/app/component/about/components/about-page/about-page.component.html [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
-  <div class="vision">
-    <div class="container-vision">
-      <img src="assets/img/Vision.png" alt="vision">
+<main id="main-content" role="main">
+  <div class="page-wrapper">
+    <div class="container-about">
       <div class="full-text-block">
         <h2>
-          {{ 'about-us.intro.block-2.header' | translate }}
+          {{ "about-us.intro.block-1.header" | translate }}
```
Comment (luchward) [inline src/app/component/about/components/about-page/about-page.component.html:6]: Why did you use double quotes in this row only? We use single quotes in interpolation.

File: src/app/component/about/components/about-page/about-page.component.html [hunk 2]
```diff
@@ -53,97 +54,87 @@ <h3 class="card-title-text">{{ 'about-us.steps.block-1.header' | translate }}</h
               <span class="main-card-text">
                 {{ 'about-us.steps.block-1.text' | translate }}
               </span>
-              <a class="card-link"
-                 [routerLink]="['/map']"
-                 fragment="top-user-bar"
-                 routerLinkActive="active-link"
-                 [routerLinkActiveOptions]="{ exact: true }">
+              <a class="card-link" [routerLink]="['/map']" fragment="top-user-bar" routerLinkActive="active-link"
```
Comment (luchward) [inline src/app/component/about/components/about-page/about-page.component.html:57]: According to the style guide, when a tag has several attributes, it is more readable to write them one per line.

File: src/app/component/about/components/about-page/about-page.component.html [hunk 3]
```diff
@@ -53,97 +54,87 @@ <h3 class="card-title-text">{{ 'about-us.steps.block-1.header' | translate }}</h
...
+                {{ 'about-us.steps.block-5.button' | translate }}
+              </a>
+            </div>
           </div>
         </div>
       </div>
     </div>
   </div>
-</div>
+</main>
```
Comment (luchward) [inline src/app/component/about/components/about-page/about-page.component.html:140]: Always leave one empty line at the end of file.

File: src/app/app.component.scss [hunk 4]
```diff
@@ -17,3 +17,21 @@
...
+.skip-container {
+  text-align: center;
+}
+
+.skip {
+  position: absolute;
+  top: 8%;
+  left: -100%;
+  width: 100px;
+  color: #13aa57;
```
Comment (luchward) [inline src/app/app.component.scss:30]: Execute:
git pull
git merge dev
and use css variables.

File: src/app/component/about/components/about-page/about-page.component.html (hunk 1 above)
Comment (KirillAbramo) [reply src/app/component/about/components/about-page/about-page.component.html:6]: Actually I didn't.. these are not my changes.

File: src/app/component/about/components/about-page/about-page.component.html (hunk 2 above)
Comment (KirillAbramo) [reply src/app/component/about/components/about-page/about-page.component.html:57]: Thanks, take it into account but this is not my code change.

File: src/app/component/about/components/about-page/about-page.component.html (hunk 3 above)
Comment (KirillAbramo) [reply src/app/component/about/components/about-page/about-page.component.html:140]: got it

**Comment by luchward** [comment]
You still leave some files without empty lines at the end.

**Review by luchward — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

**Review by ZooMingGG — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Pull request merged**